x-pack/test/api_integration/apis/security_solution/timeline_migrations.ts

```diff
@@ -28,23 +38,22 @@ export default function ({ getService }: FtrProviderContext) {
     const es = getService('es');
 
     describe('7.16.0', () => {
-      describe('notes timelineId', () => {
-        before(async () => {
-          await esArchiver.load(
-            'x-pack/test/functional/es_archives/security_solution/timelines/7.15.0'
-          );
-        });
-
-        after(async () => {
-          await esArchiver.unload(
-            'x-pack/test/functional/es_archives/security_solution/timelines/7.15.0'
-          );
-        });
+      before(async () => {
```

Comment:
Just moved these before and after functions to the top level because each of the 7.16.0 tests is using the same es archive.